dotCMS/src/main/java/com/dotmarketing/portlets/workflows/actionlet/CheckoutContentActionlet.java

```diff
@@ -29,16 +28,21 @@ public String getHowTo() {
 		return "This actionlet will checkout and lock the content.";
 	}
 
-	public void executeAction(WorkflowProcessor processor,Map<String,WorkflowActionClassParameter>  params) throws WorkflowActionFailureException {
+	public void executeAction(final WorkflowProcessor processor,
+							  final Map<String,WorkflowActionClassParameter>  params) throws WorkflowActionFailureException {
+
 		try {
 
-			APILocator.getContentletAPI().lock(processor.getContentlet(), processor.getUser(), true);
+			final Contentlet contentlet = processor.getContentlet();
+			contentlet.setProperty(Contentlet.DO_REINDEX, Boolean.FALSE);
+
+			APILocator.getContentletAPI().lock(processor.getContentlet(),
+					processor.getUser(), true);
 		} catch (Exception e) {
+
 			Logger.error(this.getClass(),e.getMessage(),e);
```

Comment:
Please include in the error message the Identifier or useful information about the Contentlet involved in the process.